.buildkite/pipeline.yml

```diff
@@ -154,54 +154,6 @@ steps:
           run: android-ci
     command: './gradlew test'
 
-  - label: ':android: Android 7 Instrumentation tests'
-    key: 'core-instrumentation-tests'
-    depends_on:
-      - "android-ci"
-    timeout_in_minutes: 60
-    command: ./scripts/build-instrumentation-tests.sh && ./scripts/run-instrumentation-test.sh
-    plugins:
-      - docker-compose#v3.7.0:
-          run: android-ci
-    env:
-      SINGLE_RUNNER: "true"
-      INSTRUMENTATION_DEVICES: '["Google Pixel 2-8.0"]'
-      TEST_APK_LOCATION: 'bugsnag-android-core/build/outputs/apk/androidTest/debug/bugsnag-android-core-debug-androidTest.apk'
-    concurrency: 9
-    concurrency_group: 'browserstack-app'
-
-  - label: ':android: NDK Instrumentation tests'
-    key: 'ndk-instrumentation-tests'
-    depends_on:
-      - "android-ci"
-    timeout_in_minutes: 60
-    command: ./scripts/build-instrumentation-tests.sh && ./scripts/run-instrumentation-test.sh
-    plugins:
-      - docker-compose#v3.7.0:
-          run: android-ci
-    env:
-      SINGLE_RUNNER: "true"
-      INSTRUMENTATION_DEVICES: '["Google Pixel 2-8.0"]'
-      TEST_APK_LOCATION: 'bugsnag-plugin-android-ndk/build/outputs/apk/androidTest/debug/bugsnag-plugin-android-ndk-debug-androidTest.apk'
-    concurrency: 9
-    concurrency_group: 'browserstack-app'
-
-  - label: ':android: Performance benchmarks'
```

Comment:
This is my bad as I forgot to mention, but we'll probably want to keep running the performance benchmarks on BrowserStack in addition to an emulator, so that the results are realistic.

Although having said that, there's definitely an argument for running them on a full build only or omitting them entirely - as so far we've just tended to run the benchmarks ourselves on the conclusion of a big project.

Reply:
I think I like the idea of just running benchmarking tests ourselves when we feel it's appropriate.  Having a human run and analyse performance test results allow us to be more subjective and explore areas in more detail if needed.  Having the tests run in CI could give a false sense of security whilst also using a limited resource.

Reply:
Sounds good to me 👍